template/base/base.html

```diff
@@ -147,11 +133,11 @@
                     </a>
 
                     <ul class="dropdown-menu dropdown-menu-right">
+                        <!--
                         <li><a href="#"><i class="icon-user-plus"></i> My profile</a></li>
-                        <li><a href="#"><i class="icon-coins"></i> My balance</a></li>
-                        <li><a href="#"><span class="badge badge-warning pull-right">58</span> <i class="icon-comment-discussion"></i> Messages</a></li>
                         <li class="divider"></li>
                         <li><a href="#"><i class="icon-cog5"></i> Account settings</a></li>
+                        -->
```

Comment:
another change here but commented out. 
so the right up corner drop down menu does not work for now?

Reply:
It currently does not connect to anything - will uncomment once it's linked to new content